docker/global-metrics/data/exporter.toml

```diff
@@ -0,0 +1,12 @@
+max_mind_db_path = "/var/lib/GeoIP/GeoLite2-City.mmdb"
+
+name = "ursa"
+protocol_name = "/ursa/kad/0.0.1"
+bootnodes = [
+  "/dns/bootstrap-node-0.ursa.earth/tcp/6009/p2p/12D3KooWDji7xMLia6GAsyr4oiEFD2dd3zSryqNhfxU3Grzs1r9p"
+]
```

Comment:
Also happy to deploy and maintain a Kademlia Exporter instance on kademlia-exporter.max-inden.de/ scraping the ursa network. It is just a small configuration change on my end. Is the above bootstrap node address stable?

Reply:
Would be awesome to see! Thanks for the comments. And yeah, the address above is stable if you do run your own